aws/kubernetes/Detailed_setup_guide.md

```diff
@@ -0,0 +1,97 @@
+
+
+### [Step 4: Create the cluster and get Kubernetes credentials](#anchor-create-eks-cluster)
+
+Once you verify that you are allowed to use the necessary VM types in your preferred location, now it's time to create a managed Kubernetes cluster, namely an EKS cluster. The process is pretty simple with `eksctl`. After you successfully deploy a cluster with a node-group of some nodes, you will be able to run workers as pods on the Kubernetes cluster using [dask-kubernetes](https://github.com/dask/dask-kubernetes).
+
+- Lets first create a minimal yaml configuration file with name `eksctl_config.yaml` with the following in it: 
+    ```yaml
+    apiVersion: eksctl.io/v1alpha5
+    kind: ClusterConfig
+
+    metadata:
+      name: <name set in EKS_CLUSTER_NAME>
+      region: <region set in REGION_NAME>
+      version: "1.20"
+
+    managedNodeGroups:
+    - name: nodegroup-managed-gpu-dev
+      minSize: 0
+      desiredCapacity: 2
+      maxSize: 4
+      instanceType: <instance type set in INSTANCE_TYPE>
+      ssh:
+        allow: true
+    ```
+    To make things simple, we will use a [managed node group](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html) to let EKS perform the provisioning and lifecycle management on our behalf. We are also allowing `ssh` access into the nodes for debugging. By default this will use the `~/.ssh/id_rsa.pub` file in your local machine. 
+
+    **NOTE:** TO use RAPIDS, you must use a GPU enabled instance in AWS. 
+
+- With the `yaml` file, creating the EKS cluster is as simple as running the following command: 
+    ```bash
+    eksctl create cluster -f eksctl_config.yaml
+    ```
+    This will take a few minutes before it completes. Grab a coffee :coffee: :coffee: . For more `yaml` configurations, you can refer to [eksctl documentation](https://eksctl.io/usage/schema/).
+
+- You can check the whether the cluster is successfully created with following:
+    ```bash
+    eksctl get cluster --name $EKS_CLUSTER_NAME --region $REGION_NAME
+    ```
+    **NOTE:** For more `eksctl` related references on how to delete or modify the cluster, visit https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html.
+
+- Once the cluster is created successfully, let's get the credentials for your EKS cluster to access it from your machine. 
+    ```bash
+    aws eks update-kubeconfig --name $EKS_CLUSTER_NAME  --region $REGION_NAME
+    ```
+- Check whether you are able to access the nodes: 
+    ```bash
+    kubectl get nodes
+
+    NAME                            STATUS     ROLES    AGE   VERSION
+    ip-172-31-12-100.ec2.internal   NotReady   <none>   10m   v1.20.4-eks-6b7464
+    ip-172-31-34-168.ec2.internal   NotReady   <none>   10m   v1.20.4-eks-6b7464
+    ```
+
+### [Step 5: Set up the EKS cluster to use GPUs for our workload](#anchor-setup-gpu)
```

Comment:
Should we add a note at the end about delete the cluster during clean-up? Or is cluster.close() enough?

`eksctl delete cluster --region=$REGION_NAME --name=$EKS_CLUSTER_NAME`

Reply:
Good idea. Lets add a subsection called cluster cleanup and add this command.